pkgs/tools/security/iaito/default.nix

```
@@ -48,14 +56,15 @@ stdenv.mkDerivation rec {
   };
 
   meta = with lib; {
-    description = "Official frontend of radare2";
+    description = "An official graphical interface of radare2";
     longDescription = ''
-      The official graphical interface for radare2, a libre reverse engineering
-      framework.
+      iaito is the official graphical interface of radare2. It's the
+      continuation of Cutter for radare2 after the Rizin fork.
     '';
-    homepage = "https://github.com/radareorg/iaito";
+    homepage = "https://rada.re/n/iaito.html";
```

We should link directly to the redirect's location.

```suggestion
    homepage = "https://radare.org/n/iaito.html";
```